src/components/AddBareMetalHosts/OCMClusterDetailTabContent.tsx

```diff
@@ -1,25 +1,50 @@
 import React from 'react';
-import { Modal, ModalVariant } from '@patternfly/react-core';
 import { normalizeClusterVersion } from '../../config';
 import { OcmClusterType } from './types';
 import { AddHostsClusterCreateParams, Cluster, getCluster, handleApiError } from '../../api';
-import RedirectToCluster from '../ui/RedirectToCluster';
 import { getOcmClusterId } from './utils';
 import { usePullSecretFetch } from '../fetching/pullSecret';
 import { ErrorState, LoadingState } from '../ui';
 import { addHostsClusters } from '../../api/addHostsClusters';
+import { AlertsContextProvider } from '../AlertsContextProvider';
+import { POLLING_INTERVAL } from '../../config';
+import AddBareMetalHosts from './AddBareMetalHosts';
 
-const AddBareMetalHostsDialog: React.FC<{
-  isOpen: boolean;
-  closeModal: () => {};
-  cluster: OcmClusterType;
-}> = ({ isOpen, cluster, closeModal }) => {
+type ClusterDetailTabType = {
+  key: number;
+  title: string;
+  contentId: string;
+  id: string;
+  show: boolean;
+  ref: React.RefObject<unknown>;
+};
+
+export const getOCMClusterDetailTab = (
```

Comment:
is this required to be in facet-lib? I feel like this is very specific to OCM tabs implementation so it should live there.

Reply:
All of these components are OCM specific and will most probably not used anywhere else. Having it here, it is easier for us to update if needed. Moreover, it conforms the tab-component.